### PR TITLE
Feature/mc 10263 delete subnet of vn

### PR DESCRIPTION
**Pull request by lamminade**

### CODE REVIEW

File: source/includes/azure/_security_rules.md [hunk 1]
```diff
@@ -65,3 +65,26 @@ Attributes | &nbsp;
 `sourceAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
 `destinationPortRanges`<br/> *List* | This specifies on which ports traffic will be allowed or denied by this rule. If the list is empty then all values are included.
 `destinationAddressPrefixes`<br/> *List* | List of IP address ranges or/and IP adresses. If the list is empty then all values are included.
+
+#### Get a security rule
```
Comment (evanluc) [inline source/includes/azure/_security_rules.md]: Did you add this or is this from someone else?

Comment (lamminade) [inline source/includes/azure/_security_rules.md]: This stuff is from PR #100  (merge this branch after that one)

Comment (lamminade) [reply source/includes/azure/_security_rules.md]: yeah this is from PR #100, merging this branch after that one ideally, that one has an update that github isnt showing for some reason

**Review by GuillaumeVialle — APPROVED** [review]
Make sure it doesn't mess up the security rule file now that I merged but looks good!!

**Pull request merged**